Java/LongestIncreasingSubsequence.java

```diff
@@ -0,0 +1,43 @@
+ /*
+ * Given a sequence A of size N, find the length of the longest increasing subsequence from a given sequence.
+ * The longest increasing subsequence means to find a subsequence of a given sequence in which the 
+ * subsequence's elements are in sorted order, lowest to highest, and in which the subsequence is as 
+ * long as possible. This subsequence is not necessarily contiguous, or unique.
+ */
+
+package Hacktoberfest2020_.Java;
+
+public class LongestIncreasingSubsequence {
+	public static class LIS_Memoization {
+		public static int lis(final int arr[]) {
+            final int n = arr.length;
+            final int lis[] = new int[n];
```

Comment:
Please follow camelCase naming conventions.

Reply:
Sure. I'll change that and submit the code again.